.github/workflows/test-analyze.yml

```
@@ -0,0 +1,92 @@
+# ==================================================
+# This is the main worflow for testing and analyzing 
+# the code.
+# ==================================================
+
+name: CI
+
+on:
+  push:
+    branches: [master, development]
```

```suggestion
    branches: [project/**, release/**]
```
We don't have development or master branch.